bounties_api/std_bounties/tests.py

```diff
@@ -1,6 +1,98 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from functools import partial
+
 from django.test import TestCase
 
 # Create your tests here.
+from std_bounties.utils import narrower, formatter, merge, pipe
+from std_bounties.models import Bounty, Fulfillment
+
+
+class BountySubscriberTest(TestCase):
```

Comment:
Thanks so much for introducing tests. Really appreciate it.

Reply:
This is going to be a good intro - I'll set up circle after this is merged, and put a bounty up to increase code coverage.